pyvo/make_vid.py

```diff
@@ -230,4 +232,10 @@ def make_pyvo(
     os.link(result.filename, outname)
     print('Saved as {}'.format(outname))
 
+    metadata = {'speaker': speaker, 'title': title, 'date': date, 'event': event,
+                'lighting': lightning, 'url': url}
+    metaname = outname[:-3] + "yaml"
```

Comment:
Why `[:-3]`? What if something's saved as `.mpeg`?
Please either use [`os.path.splitext`](https://docs.python.org/3/library/os.path.html#os.path.splitext), or only append the extension (e.g. `video.mkv.yaml`).